tests/func-tests/common/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  AlchemyProvider,
+  JsonRpcProvider,
+  TransactionReceipt,
+  TransactionResponse,
+} from '@ethersproject/providers';
+
+// todo: move to common
+export function getEnv(name: string, defaultValue?: string): string {
+  const value = process.env[name];
+
+  if (value !== undefined) {
+    return value;
+  } if (defaultValue !== undefined) {
+    return defaultValue;
+  }
+  throw new Error(`Environment variable '${name}' not set`);
+}
+
+export const env = {
+  network: getEnv('NETWORK'),
+  alchemyApiKey: getEnv('TEST_ALCHEMY_API_KEY'),
+  // client: {
+  //   publicApiUrl: getEnv('PUBLIC_API_URL'),
+  //   starkContractAddress: getEnv('TEST_STARK_CONTRACT_ADDRESS'),
+  //   registrationContractAddress: getEnv('TEST_REGISTRATION_CONTRACT_ADDRESS'),
+  // },
+  privateKey1: getEnv('TEST_PRIVATE_KEY1'),
+  starkPrivateKey1: getEnv('TEST_STARK_PRIVATE_KEY1'),
+  // privateKey2: getEnv('TEST_PRIVATE_KEY2'),
+  privateKeyBanker: getEnv('TEST_PRIVATE_KEY_BANKER'),
+  starkPrivateKeyBanker: getEnv('TEST_STARK_PRIVATE_KEY_BANKER'),
+  tokenAddress: getEnv('TEST_TOKEN_ADDRESS'),
+  // moonpayWebhookKey: getEnv('MOONPAY_WEBHOOK_KEY'),
+  // starkExBatchSize: parseInt(getEnv('TEST_STARKEX_BATCH_SIZE'), 10),
+  // unregisteredUserPrivateKey: getEnv('TEST_UNREGISTERED_USER_PRIVATE_KEY'),
+  // unregisteredUserStarkPrivateKey: getEnv('TEST_UNREGISTERED_USER_STARK_PRIVATE_KEY'),
```

Comment:
any reason these are commented out rather than delete? if someone wants to use them, can they just uncomment and use?

Reply:
They were not needed for the current tests to pass. My assumption is that they will be needed once the remaining tests are ported across.